rest/specs/src/main/filtered-resources/openapi/openapi.yaml

```diff
@@ -22,12 +22,12 @@ servers:
   - url: ${rest.server.url}
 
 paths:
-  /product:
+  /dataset:
     post:
-      summary: Create a Product
+      summary: Create a dataset
       description: >
-        Clients send a file attachment to create a Product resource
-      operationId: storeProduct
+        Clients send a file attachment to create a dataset.
+      operationId: createDataset
       tags:
         - store
```

Comment:
should we also update the tag to `create`?

Reply:
I'm not sure, but I believe the tag is the API name: https://swagger.io/docs/specification/grouping-operations-with-tags/.